install.sh

```diff
@@ -355,7 +355,7 @@ else
   git checkout "${BRANCH}"
 
   # Set sparse-checkout folders.
-  git sparse-checkout set packages lib bin crew tools
+  git sparse-checkout set packages "manifest/${ARCH}" lib bin crew tools
```

Comment:
This won't work on `aarch64` machines, btw. Better just to use `manifest` and then we can move back to `"manifest/${ARCH}"` when we support `aarch64` fully.

Reply:
It makes sense to use `ARCH` here.  We don't need to be concerned about manifest files that are not matching the architecture.  I believe @satmandu has already added logic that determines if lib64 is needed for aarch64 or not.

Reply:
The problem is armv7l installs on aarch64 userspaces-- those will need manifests in manifest/armv7l but will only try and checkout a manifest/aarch64 directory.

Reply:
Currently, we're mapping aarch64 to armv7l so that would never happen.

Reply:
What do you mean?

Reply:
I suppose we don't have a symlink from `metatag/armv7l` to `metatag/aarch64` yet so I guess you are right.
```
ARCH = KERN_ARCH.eql?('armv8l') ? 'armv7l' : KERN_ARCH
```
This logic in const.rb does change for armv8l but not for aarch64.

Reply:
And at any rate that logic is in crew-- the ARCH variable being drawn from here is the output of uname -m, in the installer.

Reply:
See #8427.